src/components/BackupUI.tsx

```diff
@@ -129,6 +130,7 @@ export function BackupUIView ({
   blobs,
   prefsDoc
 }: BackupUIViewProps) {
+  const [isModalOpen, setIsOpen] = useState<boolean>(false)
```

Comment:
let's keep the names sync'd
```suggestion
  const [isModalOpen, setIsModalOpen] = useState<boolean>(false)
```